openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Tag.spec.ts

```diff
@@ -313,6 +313,79 @@ test.describe('Tag Page with Admin Roles', () => {
       dataTestId: 'owner-link',
     });
   });
+
```

Comment:
Let's also add a test for disabled tags.
Visit disabled tag page and verify we shouldn't be able to attach asset from "Assets" tab

Reply:
>Another test i can think of, make tags disabled and go to table page and check it should not appear in tags component.

This already exists https://github.com/open-metadata/OpenMetadata/pull/18242